<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDelivery</class>
 <widget class="QWidget" name="AddDelivery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#AddDelivery {
    background-color: #f0f0f0;
}

QLabel {
    font-size: 10px;
}

QLineEdit, QTableView {
    background-color: #ffffff;
    border: 1px solid #bcbcbc;
    padding: 5px;
    margin-bottom: 10px;
}

QPushButton {
    background-color: #3498db;
    color: #ffffff;
    border: none;
    border-radius: 3px;
}

QPushButton:hover {
    background-color: #45a049;
}
</string>
  </property>
  <widget class="QLabel" name="OrderIDLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
font-size: 10px;
}</string>
   </property>
   <property name="text">
    <string>Order ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="ProductIDLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
font-size: 10px;
}</string>
   </property>
   <property name="text">
    <string>Product:</string>
   </property>
  </widget>
  <widget class="QLabel" name="CustomerLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
font-size: 10px;
}</string>
   </property>
   <property name="text">
    <string>Customer:</string>
   </property>
  </widget>
  <widget class="QLabel" name="AmountLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
font-size: 10px;
}</string>
   </property>
   <property name="text">
    <string>Amount:</string>
   </property>
  </widget>
  <widget class="QLabel" name="AddressLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
font-size: 10px;
}</string>
   </property>
   <property name="text">
    <string>Delivery Address:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="OrderIDLineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ProductLineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="AmountLineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="CustomerLineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddressLineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>11</y>
     <width>371</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="LoadTableButton">
   <property name="geometry">
    <rect>
     <x>345</x>
     <y>192</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddDeliveryButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>192</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Delivery Order</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
